code/upaya/app/helpers/idp_helper.rb

```diff
@@ -0,0 +1,5 @@
+module IdpHelper
+  def get_help_image_uri(question)
+    "help/#{File.basename(question['helpImageUrl'])}"
+  end
+end
```

Comment:
This isn't related to authentication. If we also want to import the Id Verification code from ICAM, what do you think about doing it in a separate PR, and renaming IdP to IdV in the cases where it refers to Id Verification (as opposed to Identity Provider).